### DISCUSSION TITLE
Error in foundary installation

### DISCUSSION THREAD

**Discussion by ObedCyber** [Cyfrin/foundry-full-course-cu#1016] [General]
I tried to install foundary from the official docs with this code
```bash
curl -L https://foundry.paradigm.xyz | bash
```
 but I'm getting this error 
curl: (35) error:0A000126:SSL routines::unexpected eof while reading How do i fix this

**Comment by theirrationalone** [comment]
Experiments to mitigate this....

1. User `VPN` and try again.
or
2. install `openssh` and try again.

**Comment by mahantybiplab** [comment]
Use vpn , it worked for me
